elisp/intero.el

```diff
@@ -1261,11 +1258,11 @@ Automatically performs initial actions in SOURCE-BUFFER, if specified."
                       (when intero-debug
                         (message "Intero arguments: %s" (combine-and-quote-strings arguments)))
                       (message "Booting up intero ...")
-                      (apply #'start-process "stack" buffer "stack" "ghci"
-                             arguments))))
+                      (get-buffer-process (apply #'make-comint-in-buffer "stack" buffer "stack" nil "ghci"
```

Comment:
This one can stay as `start-process` because it's not actually a comint buffer, it's used in the background for the backend stuff.